### PR TITLE
Add basic hotkeys

### LINKED ISSUE #56: Support most basic hotkeys and convenience functionalities in monaco
- ctrl-c (how does this work for multiple selections?)
- ctrl-a
- ctrl-v (how does this work for multiple selections/cursors?)
- delete selected range (multiple selections should be feasible too)

### PR DESCRIPTION
**Pull request by katzper01**
Resolves #56

### CODE REVIEW

File: src/components/editor/latex-textarea/MonacoContentManager.ts [hunk 1]
```diff
@@ -93,33 +93,59 @@ export class MonacoContentManager {
...
+    isBackspace: boolean,
     generateCharacter: (val: string) => Character
-  ): Delta | undefined {
-    const { lineNumber: row, column } = editorRef.current.getPosition();
-    const currentOffset = this.positionToOffset({row, column});
-    const prevId = (currentOffset !== -1) ? this.undeletedDocument.at(currentOffset).id : null;
+  ) : Delta[] {
+    // Trim inserted text to prevent Ulysses pasting incident.
+    insertedText = (insertedText.length > 24) ? 
+      insertedText.substring(0, 24) : insertedText;
```
Comment (hubzie) [inline src/components/editor/latex-textarea/MonacoContentManager.ts:105]: Extract constant.

Also I think 24 is definitely too little, 128 maybe?

**Review by hubzie — APPROVED** [review]
I didn't even suppose that you can implement animated copy&paste. It's not a bug, it's a feature.

PS. `Ctrl+X` works for me as well

File: src/components/editor/latex-textarea/MonacoContentManager.ts (hunk 1 above)
Comment (katzper01) [reply src/components/editor/latex-textarea/MonacoContentManager.ts]: moved to constant and set 128 for now

File: src/pages/main/collaboration/delta-queue/delta-queue.ts [hunk 2]
```diff
@@ -42,6 +45,12 @@ export const useDeltaQueue = (
   }, [state.deltaQueue]);
 
   const push = (delta: Delta) => {
+    if (isDeltaApplied(delta)) {
+      return;
+    }
+
+    console.log(JSON.stringify(delta));
```
Comment (hubzie) [inline src/pages/main/collaboration/delta-queue/delta-queue.ts:52]: Remove log (or make it debug level) - adapt it across whole PR

**Review by hubzie — APPROVED** [review]
LGTM :+1:

**Pull request merged**